### DISCUSSION TITLE
Updating from Big Sur to Monterey

### DISCUSSION THREAD

**Discussion by arunanosaurus** [SkyrilHD/Dell-Exx50-Hackintosh#41] [Q&A]
Hey Skyril,

First off thanks for your directions, I was able to successfully install Big Sur with everything working except for (Airdrop, and bluetooth) on the E7450, using your EFI folder. It was my first hackintosh project and it turned out well.

I now have an update for Monterey show up on system preferences. Should I go ahead and update it directly? or a bootable disk is recommended? Quite new to this, so your guidance will help me a lot. Thank you.

**Comment by SkyrilHD** [comment]
You should be able to update without any issue, but you need to update to v2.0 as it has fixes for Monterey in general. But I will release OpenCore 0.7.5 soon with some minor fixes as well.
